diagnostics/surface_albedo_feedback/settings.jsonc

```diff
@@ -0,0 +1,44 @@
+// Example MDTF diagnostic settings file
+//
+// This should be an ascii text file in JSON format 
+// (https://en.wikipedia.org/wiki/JSON#Data_types_and_syntax), with the addition
+// that all text to the  right of a "//" is a comment and ignored (sometimes 
+// called "JSONC" format).
+// 
+{
+  "settings" : {
+    "driver" : "surface_albedo_feedback.py",
+    "long_name" : "Surface Albedo Feedback Diagnostics",
+    "realm" : "atmos",
+    "description" : "Surface Albedo Feedback Diagnostics",
+    "pod_env_vars" : { "TAS_var": "tas"},
```

Comment:
`tas` should be specified in the varlist section as a requirement or optional variable, depending on how the POD uses it.